flask_appbuilder/security/manager.py

```diff
@@ -261,11 +263,27 @@ def __init__(self, appbuilder):
                 self.oauth_remotes[provider_name] = obj_provider
 
         # Setup Flask-Login
+        self.set_login_manager(app)
+
+        # Setup Flask-Jwt-Extended
+        self.set_jwt_manager(app)
+
+    def set_login_manager(self, app):
```

Comment:
Alternate naming / strategy:
```
def build_login_manager(self, app): -> LoginManager
```
This allows the `lm` to remain "private" and still be fully customizable